src/js/components/dialog.js

```diff
@@ -82,8 +82,10 @@ export default class Dialog extends Component {
       _initElements () {
         const dialogId = this.element.getAttribute(this.dialogAttribute)
 
-        this.triggerButton = document.querySelector(`[${this.triggerAttribute} = "${dialogId}"]`)
+        this.triggerButtons = document.querySelectorAll(`[${this.triggerAttribute} = "${dialogId}"]`)
```

Comment:
Can we wrap this `document.querySelectorAll(...)` in an `Array.from()`? Babel won't transpile a `.forEach` loop on a NodeList because forEach is implemented in all browser on Arrays, but not necessarily implement on the NodeList. It would be a bit safer here since we're using .forEach on this.triggerButtons later on to add the event listeners.

```js
this.triggerButtons = Array.from(
  document.querySelectorAll(`[${this.triggerAttribute} = "${dialogId}"]`)
)
```